test/production/app-dir/metadata-streaming-config/metadata-streaming-config-customized.test.ts

```diff
@@ -1,5 +1,8 @@
 import { nextTestSetup } from 'e2e-utils'
 
+// TODO: remove this env once streaming metadata is available for ppr
+process.env.__NEXT_EXPERIMENTAL_PPR = 'true'
```

Comment:
I think we can omit these from the tests -- the PPR runner will already run them with the flag turned on.

Reply:
It was also for local tests and the non-PPR tests on CI, if we don't set it won't work. I was thinking to make it work like other normal PPR specific tests where it should pass without manually passing `__NEXT_EXPERIMENTAL_PPR`